float64[18] q_target
float64[18] qd_target
float64[18] trq_target
float64 q_first_flag